prow/plugins/override/override.go

```diff
@@ -135,6 +155,70 @@ func authorized(gc githubClient, log *logrus.Entry, org, repo, user string) bool
 	return ok
 }
 
+func authorizedTopLevelOwner(oc ownersClient, allowTopLevelOwners bool, log *logrus.Entry, org, repo, user string, pr *github.PullRequest) bool {
+	if allowTopLevelOwners {
+		owners, err := oc.LoadRepoOwners(org, repo, pr.Base.Ref)
+		if err != nil {
+			log.WithError(err).Warnf("cannot determine whether %s is a top level owner of %s/%s", user, org, repo)
+			return false
+		}
+
+		if owners.TopLevelApprovers().Has(user) {
```

Comment:
nit: this could be

```go
return owners.TopLevelApprovers().Has(user)
```

Reply:
Thank you for catching that. Updated the PR.